tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -23,6 +23,29 @@ def test_mpc_tensor_exception(get_clients) -> None:
         x = MPCTensor(secret=torch.Tensor([1, -2]), session=session)
 
 
+def test_mpc_tensor_nosession_exception(get_clients) -> None:
+    alice_client, bob_client = get_clients(2)
+
+    with pytest.raises(ValueError):
+        MPCTensor(secret=42)
+
+    with pytest.raises(ValueError):
+        x = MPCTensor(secret=torch.Tensor([1, -2]))
+
+
+def test_mpc_share_nosession_exception(get_clients) -> None:
+    alice_client, bob_client = get_clients(2)
```

Comment:
MAYBE: Remove this

Reply:
Remove what exactly?

Reply:
Q: @kamathhrishi I am not sure if you saw this, I think we can remove it